examples/game/breakout.rs

```diff
@@ -158,79 +220,26 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
         ..default()
     });
 
-    // left
-    commands
-        .spawn_bundle(SpriteBundle {
-            transform: Transform {
-                translation: Vec3::new(-PLAY_AREA_BOUNDS.x / 2.0, 0.0, 0.0),
-                scale: Vec3::new(WALL_THICKNESS, PLAY_AREA_BOUNDS.y + WALL_THICKNESS, 1.0),
-                ..default()
-            },
-            sprite: Sprite {
-                color: WALL_COLOR,
-                ..default()
-            },
-            ..default()
-        })
-        .insert(Collider);
-    // right
-    commands
-        .spawn_bundle(SpriteBundle {
-            transform: Transform {
-                translation: Vec3::new(PLAY_AREA_BOUNDS.x / 2.0, 0.0, 0.0),
-                scale: Vec3::new(WALL_THICKNESS, PLAY_AREA_BOUNDS.y + WALL_THICKNESS, 1.0),
-                ..default()
-            },
-            sprite: Sprite {
-                color: WALL_COLOR,
-                ..default()
-            },
-            ..default()
-        })
-        .insert(Collider);
-    // bottom
-    commands
-        .spawn_bundle(SpriteBundle {
-            transform: Transform {
-                translation: Vec3::new(0.0, -PLAY_AREA_BOUNDS.y / 2.0, 0.0),
-                scale: Vec3::new(PLAY_AREA_BOUNDS.x + WALL_THICKNESS, WALL_THICKNESS, 1.0),
-                ..default()
-            },
-            sprite: Sprite {
-                color: WALL_COLOR,
-                ..default()
-            },
-            ..default()
-        })
-        .insert(Collider);
-    // top
-    commands
-        .spawn_bundle(SpriteBundle {
-            transform: Transform {
-                translation: Vec3::new(0.0, PLAY_AREA_BOUNDS.y / 2.0, 0.0),
-                scale: Vec3::new(PLAY_AREA_BOUNDS.x + WALL_THICKNESS, WALL_THICKNESS, 1.0),
-                ..default()
-            },
-            sprite: Sprite {
-                color: WALL_COLOR,
-                ..default()
-            },
-            ..default()
-        })
-        .insert(Collider);
+    // Walls
+    commands.spawn_bundle(WallBundle::new(WallLocation::Left));
+    commands.spawn_bundle(WallBundle::new(WallLocation::Right));
+    commands.spawn_bundle(WallBundle::new(WallLocation::Bottom));
+    commands.spawn_bundle(WallBundle::new(WallLocation::Top));
 
-    // Add bricks
-    let bricks_width = BRICK_COLUMNS as f32 * (BRICK_SIZE.x + BRICK_SPACING) - BRICK_SPACING;
+    // Bricks
+    let brick_width = BRICK_COLUMNS as f32 * (BRICK_SIZE.x + BRICK_SPACING) - BRICK_SPACING;
```

Comment:
This rename seems slightly less clear to me.

Reply:
It's the width of a single brick, not all the bricks. It makes sense to me.

Reply:
It's the width of a single brick, not all the bricks. It makes sense to me.